app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>FarMarRails</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+
+  <%= image_tag("farmar.jpeg", alt: "Pretty Fruits and Vegetables", class: "bar-image") %>
+
+  <h1>Welcome to FarMar!</h1>
+  <%= link_to "HOME", root_path, class: "home" %>
+
+<%= yield %>
+
```

Comment:
This `yield` should have the same indentation as the `h1` and `link_to` tabs above it